DUI3-DX/Connectors/Autocad/Speckle.Connectors.AutocadShared/Operations/Receive/HostObjectBuilder.cs

```diff
@@ -60,20 +51,28 @@ CancellationToken cancellationToken
     // POC: Will be addressed to move it into AutocadContext!
     using (TransactionContext.StartTransaction(Application.DocumentManager.MdiActiveDocument))
     {
-      foreach ((List<string> path, Base obj) in objectsWithPath)
+      foreach (TraversalContext tc in traversalGraph)
       {
         cancellationToken.ThrowIfCancellationRequested();
 
         try
         {
-          string layerFullName = _autocadLayerManager.LayerFullName(baseLayerPrefix, string.Join("-", path));
+          string layerFullName = GetLayerPath(tc, baseLayerPrefix);
 
           if (uniqueLayerNames.Add(layerFullName))
           {
             _autocadLayerManager.CreateLayerOrPurge(layerFullName);
           }
 
-          object converted = _converter.Convert(obj);
+          //POC: this transaction used to be called in the converter, We've moved it here to unify converter implementation
```

Comment:
We will reconsider it on Beta phase to refactor AutocadContext with Document and Transactions. Should be safe for now to have it here.

Reply:
Thanks for investigating, 
Shall I remove this POC comment?

Reply:
You can edit it by saying: Transaction logic will be moved into AutocadContext.. soemthing like this